pkg/manager/member/scaler.go

```diff
@@ -35,6 +35,12 @@ const (
 	skipReasonScalerAnnDeferDeletingIsEmpty = "scaler: pvc annotations defer deleting is empty"
 )
 
+var (
+	hostRegionLabel = map[string]string{
+		"specialUse": "hotRegion",
```

Comment:
describe the purpose of this special label? why we need to add this special label on newly created TiKV instances. If PD has public docs about this label, we can link too.

Reply:
Nice catch, I will add `TODO` to mark there should be some docs about this label after I finish the entire auto-scaling document.

Reply:
great! comments describing the purpose or reasons behind will make it easier to understand the code.